AlfrescoApp/Config/Helpers/AlfrescoViewConfigHelper.m

```diff
@@ -212,6 +212,18 @@ - (AlfrescoGroupConfigData *)viewGroupDataFromJSON:(NSDictionary *)viewGroupJSON
     return viewGroupConfigData;
 }
 
+- (NSString *)evaluatorFor:(NSString *)viewId
+{
+    NSString *evaluator = nil;
+    NSDictionary *viewsDict = self.json[kAlfrescoJSONViews];
```

Comment:
This is fine and doesn't need updating, but generally try to avoid creating unnecessary variables where not needed - especially in code that might be called frequently.
```objectivec
NSDictionary *viewEvaluators = self.json[kAlfrescoJSONViews][viewId];
```